devito/function.py

```diff
@@ -47,6 +47,12 @@ class Constant(AbstractSymbol, ConstantArgProvider):
 
     """
     Symbol representing constant values in symbolic equations.
+
+    .. note::
+
+        The parameters must always be given as keyword arguments, since
+        SymPy uses `*args` to (re-)create the dimension arguments of the
```

Comment:
If memory serves, double ` is needed to produce correct teletype in the docs. Also below...

Reply:
fixed